dart/lib/src/sentry_measurement_unit.dart

```diff
@@ -1,4 +1,12 @@
+/// The unit of measurement of a metric value.
+/// Units augment metric values by giving them a magnitude and semantics.
+/// Units and their precisions are uniquely represented by a string identifier.
+///
+/// This is a singe enum because Enhanced Enums in Dart is only available
+/// in newer versions.
```

Comment:
Limitation before Dart 2.17 and we cannot do breaking changes just yet.
It's either this or an open String, but this has better UX, so users don't need to find out the units by themselves via docs.